macOS.md

```diff
@@ -38,6 +38,11 @@ You can close Teamviewer now :wink:
 If you are not familiar with video calls, here is a great [article](https://martinfowler.com/articles/effective-video-calls.html) full of good practices :camera: :microphone:
 
 
+## A note about quitting apps on a Mac 
+
+Clicking the little red cross in the top left corner of the application window on a Mac **does not really quit it**, it just closes an active window. To quit the application _for real_ either press `Cmd + Q` when the application is active, or navigate to APP_NAME -> Quit in the menu bar.
```

Comment:
I'm not sure people know how to locate `APP_NAME -> Quit in the menu bar`
Shall we add a small screenshot?

Reply:
Done